stable/mlrun/values.yaml

```diff
@@ -184,6 +184,90 @@ api:
       periodSeconds: 15
       failureThreshold: 4
 
+db:
+  name: db
+  fullnameOverride:
+
+  replicaCount: 1
+
+  image:
+    repository: mysql
+    tag: 5.7
```

Comment:
I'm wondering whether we should go with 5.7 or start directly with 8
@quaark / @omesser WDYT ?

Reply:
I think it's an issue regarding avrez-manifest, if we want to upgrade here to mysql 8 then we'd need to do it for pipelines as well. I'm not against it per say, but it is added thing that might cause problems if we're not careful.

Reply:
We can workaround avrez by naming the image `mlrun-mysql` or something, that's not a problem